scipy/spatial/tests/test_distance.py

```diff
@@ -609,6 +609,31 @@ def test_cdist_out(self):
             out5 = np.empty((out_r, out_c), dtype=np.int64)
             assert_raises(ValueError, cdist, X1, X2, metric, out=out5, **kwargs)
 
+    def test_striding(self):
+        # test that striding is handled correct with calls to
+        # _copy_array_if_base_present
+        eps = 1e-07
+        X1 = eo['cdist-X1'][::2, ::2]
+        X2 = eo['cdist-X2'][::2, ::2]
+        X1_copy = X1.copy()
+        X2_copy = X2.copy()
+
+        # confirm contiguity
+        assert_(not X1.flags.c_contiguous)
+        assert_(not X2.flags.c_contiguous)
+        assert_(X1_copy.flags.c_contiguous)
+        assert_(X2_copy.flags.c_contiguous)
+
+        for metric in _METRICS_NAMES:
+            kwargs = dict()
+            if metric in ['minkowski', 'wminkowski']:
+                kwargs['p'] = 1.23
+            if metric == 'wminkowski':
```

Comment:
This if statement can be indented; it'll only be true if the previous conditional was true.